src/parser/jobs/brd/modules/RefulgentProcs.tsx

```diff
@@ -23,4 +31,38 @@ export class RefulgentProcs extends Procs {
 		Avoid using actions that grant <StatusLink {...this.data.statuses.STRAIGHT_SHOT_READY} /> when you
 		could use <ActionLink {...this.data.actions.REFULGENT_ARROW} /> instead.
 	</Trans>
+
+	override output() {
+		const allOverwrites = this.getOverwritesForStatus(this.data.statuses.STRAIGHT_SHOT_READY.id).sort((a, b) => a.timestamp - b.timestamp)
+		if (allOverwrites.length === 0) { return }
+
+		return <Fragment>
+			<Trans id="brd.procs.refulgentproc.content">
+				<StatusLink {...this.data.statuses.STRAIGHT_SHOT_READY} /> is always generated by <ActionLink {...this.data.actions.BARRAGE} />&nbsp;
+				and may be generated by <ActionLink {...this.data.actions.BURST_SHOT} />, <ActionLink {...this.data.actions.IRON_JAWS} />,&nbsp;
+				<ActionLink {...this.data.actions.CAUSTIC_BITE} />, and <ActionLink {...this.data.actions.STORMBITE} />.
+			</Trans>
```

Comment:
Could you wrap this in a `<Message>` tag to maintain consistency with how other modules display their output?